src/pages/teaching-tech.js

```diff
@@ -0,0 +1,163 @@
+import React from 'react'
+
+import Layout from '../components/Layout'
+import ConstrainedWidth from '../components/Layout/ConstrainedWidth'
+import Heading from '../components/Heading'
+import Text from '../components/Text'
+import VerticalSpacing from '../components/VerticalSpacing'
+import { ExternalLink } from '../components/Link'
+import PageMeta from '../components/PageMeta'
+import Centered from '../components/Centered'
+
+const TeachingTech = () => (
+  <Layout>
+    <PageMeta
+      title="Teaching Tech"
+      description="Want to get better at helping developers learn? Join our accredited course to improve
+              your teaching skills"
+    />
+    <ConstrainedWidth>
+      <Centered>
+        <Heading level={1}>
+          <Text size="large" weight={500}>
+            Teaching Tech
+          </Text>
+        </Heading>
+        <p>
+          <Heading level={3}>
```

Comment:
The content of our other pages is in markdown files. I wonder if we should stick to that for this page also? I think the idea was that it would be easier for people to edit without breaking stuff.